src/themes/default/components/option.js

```diff
@@ -0,0 +1,60 @@
+const baseStyle = {
+  outline: 'none',
+  textDecoration: 'none',
+  display: 'flex',
+  position: 'relative',
+  backgroundColor: 'white',
+  borderBottom: (theme) => `1px solid ${theme.colors.border.muted}`,
+
+  _before: {
+    content: '""',
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    bottom: 0,
+    width: 2,
+    borderRadius: '0px 2px 2px 0px',
+    backgroundColor: 'colors.blue500',
+    transition: '0.25s',
+    transformOrigin: 'left center',
+    transform: 'scaleX(0)'
+  },
+
+  _isSelectable: {
+    cursor: 'pointer'
+  },
+
+  _hover: {
+    backgroundColor: 'colors.gray75'
+  },
+
+  _focus: {
+    backgroundColor: 'colors.gray75'
+  },
+
+  _active: {
+    backgroundColor: 'intents.info.background',
+  },
+
+  _selected: {
+    backgroundColor: 'intents.info.background',
+
+    ' span': {
```

Comment:
Should this be `& span`? IIRC it will work in glamor, but think we should be consistent.

Reply:
Also not a huge fan that we have knowledge of the component architecture in the theme.

Reply:
Agreed, we will change this in the follow up pr